manager/manager.go

```diff
@@ -101,6 +101,10 @@ const (
 
 func (s *SyncManager) Start() error {
 
+	if logUtils.IsCleanOnStartup() {
+		logUtils.CleanForStartup()
```

Comment:
rather than describing when this function runs you should name it after what it does, like `CleanupLbrynet` or something like that